Polyhedron/demo/Polyhedron/Plugins/Mesh_3/CMakeLists.txt

```diff
@@ -1,55 +1,51 @@
 include( polyhedron_demo_macros )
 
-if ( Boost_VERSION GREATER 103400 )
-  remove_definitions(-DQT_STATICPLUGIN)
-
-  qt5_wrap_cpp( VOLUME_MOC_OUTFILES ${CMAKE_CURRENT_SOURCE_DIR}/Volume_plane_thread.h )
-  qt5_wrap_cpp( VOLUME_MOC_OUTFILES ${CMAKE_CURRENT_SOURCE_DIR}/Volume_plane_interface.h )
-  qt5_wrap_ui( meshingUI_FILES  Meshing_dialog.ui Smoother_dialog.ui Local_optimizers_dialog.ui )
-  polyhedron_demo_plugin(mesh_3_plugin Mesh_3_plugin
-                         Mesh_3_plugin_cgal_code.cpp Meshing_thread.cpp split_polylines.cpp
-                         ${meshingUI_FILES})
-  target_link_libraries(mesh_3_plugin PUBLIC scene_polyhedron_item scene_polygon_soup_item scene_polylines_item scene_implicit_function_item scene_image_item
-                        scene_surface_mesh_item scene_c3t3_item ${OPENGL_gl_LIBRARY} )
-
-  set(VTK_LIBS "")
-  find_package(VTK QUIET COMPONENTS
-    vtkImagingGeneral  vtkIOImage NO_MODULE)
-  if (VTK_FOUND)
-    include(${VTK_USE_FILE})
-    if ("${VTK_VERSION_MAJOR}" GREATER "5")
-      if(VTK_LIBRARIES)
-        add_definitions(-DCGAL_USE_VTK)
-        set(VTK_LIBS vtkImagingGeneral vtkIOImage)
-      else()
-        message(STATUS "NOTICE : the DICOM files (.dcm) need VTK libraries to be open and will not be able to.")
-      endif()
+remove_definitions(-DQT_STATICPLUGIN)
+
+qt5_wrap_cpp( VOLUME_MOC_OUTFILES ${CMAKE_CURRENT_SOURCE_DIR}/Volume_plane_thread.h )
+qt5_wrap_cpp( VOLUME_MOC_OUTFILES ${CMAKE_CURRENT_SOURCE_DIR}/Volume_plane_interface.h )
+qt5_wrap_ui( meshingUI_FILES  Meshing_dialog.ui Smoother_dialog.ui Local_optimizers_dialog.ui )
+polyhedron_demo_plugin(mesh_3_plugin Mesh_3_plugin
+  Mesh_3_plugin_cgal_code.cpp Meshing_thread.cpp split_polylines.cpp
+  ${meshingUI_FILES})
+target_link_libraries(mesh_3_plugin PUBLIC scene_polyhedron_item scene_polygon_soup_item scene_polylines_item scene_implicit_function_item scene_image_item
```

Comment:
That cannot work in master, there is no more scene_polyhedron_item target

Reply:
I was wrong when I did the merge. Sorry.